src/components/Loading/Loading.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import styles from './Loading.module.scss';
+import classNames from 'classnames/bind';
+import loadingImg from '../../images/loading.gif';
+
+interface Props {
+  loading: boolean;
+  children: () => React.ReactElement;
```

Comment:
요 children을 함수로 받는 이유가 있을까요?
아래와 같이 했을 때와 어떤 차이가 있는지 궁금합니다.

```suggestion
  children: React.ReactElement;
```

Reply:
해당 `Loading` 컴포넌트는 자식컴포넌트에 추가속성값을 넘겨주는게 없기때문에 이 예에서는 동근님 말씀대로 해도 전혀 상관없습니다! 다만 과제 하다가 갑자기 `Loading`컴포넌트는 렌더속성값으로 만들어보고 싶어져서 저렇게 했습니다